src/Main.java

```diff
@@ -2809,19 +2907,19 @@ else if (level.equals("S5")) {
 				* One last regular Highjump will then reach this check.
 				*/
 				if (location == 0) {
-					return inventory.contains(Items.RUST_SPRAY) && canGP(inventory)
```

Comment:
This may have been a merge conflict gone wrong? I'm not sure why the overalls checks are being removed for the blue chest room.

Reply:
Definitely was a merge gone wrong. It was the case on a couple of other areas there somehow too. Fixed!